### PR TITLE
Remove method/sub checks, refactor tests, change perltidy options

### PR DESCRIPTION
**Pull request by m-dango**
Temporarily based on top of another branch for clarity of changes.

### CODE REVIEW

File: exercises/practice/custom-set/.meta/solutions/CustomSet.pm [hunk 1]
```diff
@@ -63,23 +59,12 @@ sub add {
...
 sub difference {
     my ( $self, $other ) = @_;
-    return __PACKAGE__->new(
-        elements => [
-            grep { $self->contains($_) && !$other->contains($_) }
-                $self->_keys,
-            $other->_keys
-        ]
-    );
+    return __PACKAGE__->new( elements => [ grep { $self->contains($_) && !$other->contains($_) } $self->_keys, $other->_keys ] );
```
Comment (glennj) [inline exercises/practice/custom-set/.meta/solutions/CustomSet.pm:68]: This does not look like an improvement

**Review by glennj — COMMENTED** [review]
Looks good.

File: exercises/practice/custom-set/.meta/solutions/CustomSet.pm (hunk 1 above)
Comment (m-dango) [reply exercises/practice/custom-set/.meta/solutions/CustomSet.pm:68]: I can likely change this formatting manually, I'm not too concerned with example solutions but its a good point nonetheless.

File: exercises/practice/series/series.t [hunk 2]
```diff
@@ -40,9 +40,7 @@ is(
 
 is(
     slices( "918493904243", 5 ),
-    [   "91849", "18493", "84939", "49390", "93904", "39042", "90424",
-        "04243"
-    ],
+    [ "91849", "18493", "84939", "49390", "93904", "39042", "90424", "04243" ],
```
[the remark this answers is not in the document]
Comment (m-dango) [reply exercises/practice/series/series.t:41]: At a glance it didn't seem worth changing for the length here, and while I don't strictly adhere to the 80 column rule, it is just below that length.

**Review by glennj — APPROVED** [review]

**Review by kotp — APPROVED** [review]

**Pull request merged**